Modules/Loadable/Colors/MRML/vtkMRMLColorBarDisplayNode.cxx

```diff
@@ -0,0 +1,240 @@
+/*==============================================================================
+
+  Program: 3D Slicer
+
+  See COPYRIGHT.txt
+  or http://www.slicer.org/copyright/copyright.txt for details.
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+==============================================================================*/
+
+#include "vtkMRMLColorBarDisplayNode.h"
+
+// MRML includes
+#include <vtkMRMLDisplayableNode.h>
+#include <vtkMRMLProceduralColorNode.h>
+#include <vtkMRMLColorTableNode.h>
+#include <vtkMRMLScene.h>
+
+// VTK includes
+#include <vtkCallbackCommand.h>
+
+// STD includes
+#include <cstring>
+
+//------------------------------------------------------------------------------
+namespace
+{
+
+const char* DISPLAYABLE_REFERENCE_ROLE = "displayableRef";
```

Comment:
no need for `displayableRef` becaues display nodes already maintain a pointer to their associated displayable node by other means

Reply:
Fixed.